dev/io.openliberty.jcache.internal/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -80,7 +80,7 @@
 	</Designate>
 
 	<OCD id="io.openliberty.jcache.cache.metatype" ibm:alias="cache"
-		name="internal" description="internal use only">
+		name="cache" description="cache.desc" ibm:beta="true">
```

Comment:
Why is this `cache` and `cache.desc` instead of `%cache` and `%cache.desc`?

Reply:
It should have the %. I'll fix this typo.